crates/edr_napi/src/solidity_tests/config.rs

```diff
@@ -104,6 +98,11 @@ pub struct SolidityTestRunnerConfigArgs {
     /// Map of RPC endpoints from chain name to RPC urls for fork cheat codes,
     /// e.g. `{ "optimism": "https://optimism.alchemyapi.io/v2/..." }`
     pub rpc_endpoints: Option<HashMap<String, String>>,
+    /// Optional RPC cache path. If this is none, then no RPC calls will be
+    /// cached, otherwise data is cached to `<rpc_cache_path>/<chain
+    /// id>/<block number>`. Caching can be disabled for specific chains
+    /// with `rpc_storage_caching`.
+    pub rpc_cache_path: Option<String>,
```

Comment:
I followed the instantiations of this member variable and it seems that this path does not include the RPC endpoint url.

In EDR we explicitly separate cache entries between different endpoint urls (e.g. providers). Is that not a concern for the test runner's caching mechanism?

Reply:
I'd guess that's not super important while we are using Foundry's EVM, and that when we replace it with EDR's EVM we'll get that behavior automatically?

Is that the case @agostbiro?

Reply:
Thanks for the explanation. Makes sense!